README.md

```diff
@@ -1 +1,87 @@
-# fuelstop-examples
+# FuelStop
+
+## Getting Started
+
+#### Projects
+
+After sign up, you will have 2 projects, one for each Ethereum network.  You can find your projects by clicking on the menu icon on the top-left corner on the page. One project is for Mainnet and the other is for Ropsten (a proof-of-work Ethereum testnet). Clicking on any of the projects will bring you to it's project details page.
+
+![menu](./diagrams/projects.svg)
+
+#### Project Details
+
+The project details page includes the project's balance, registered addresses, transaction history, account history, and unique JSON-RPC url containing the Project ID which is used to authenticate your API calls.
+
+###### Balance
+
+This section is where you can add your credit card with FuelStop, and after which displays information about the account associated with the project. The Mainnet project would start without credit card added, and therefore you have to add one before we can fund your transactions. The Ropsten project, however, is with a testnet of Ethereum, and does not require you to add any credit card to start using. It comes with a fake credit card, and you can top it up anytime you want by clicking on the "Top Up Button" with no cost. This is where you can test your system or library with FuelStop.
+
+![Balance without credit card](./diagrams/balance-without-credit-card.svg)
+
+![FuelStop Documentation](./diagrams/balance.svg)
+
+###### Details
+
+Here you can find the name of the network the project is associated with, and your RPC URL. FuelStop's Ethereum JSON-RPC APIs require a valid project ID to be part of your URL to authenticate the calls. This is how we identify who you are behind the API requests, and it is important to keep it a secret.
+
+![Details](./diagrams/details.svg)
+
+###### Addresses
+
+Before sending your signed transactions to us via `eth_sendRawTransaction`, you need to register your address with us. If you send FuelStop a signed transaction via your RPC URL that is not included in your registered addresses, FuelStop will behave no differently than a regular Ethereum node and will broadcast it to the network. If your address has some ETH but not enough to cover the transaction's fees, FuelStop will fund your address with the exact amount needed for the transaction to go through.
```

Comment:
```suggestion
Before sending your signed transactions (via the JSON-RPC `eth_sendRawTransaction` method or with a client-library), you need to register your addresses. If you send FuelStop a signed transaction that is not included in your address-list, FuelStop will behave no differently than a regular Ethereum node and will broadcast it to the network. If your address has some ETH but not enough to cover the transaction's fees, FuelStop will fund your address so that the transaction can go through.
```